DependencyInjection/SonataAdminExtension.php

```diff
@@ -90,6 +90,7 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sonata.admin.configuration.dashboard_blocks', $config['dashboard']['blocks']);
         $container->setParameter('sonata.admin.configuration.sort_admins', $config['options']['sort_admins']);
         $container->setParameter('sonata.admin.configuration.breadcrumbs', $config['breadcrumbs']);
+        $container->setParameter('sonata.admin.configuration.role_sonata_admin', $config['security']['roles']['role_sonata_admin']);
```

Comment:
Is this parameter really needed? Why not inject its value directly in the twig extension? Also, you forgot to do precisely that.